webpack/assets/javascripts/react_app/components/AutoComplete/components/AutoCompleteMenu.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import groupBy from 'lodash/groupBy';
+import { TypeAheadSelect } from 'patternfly-react';
+import SubstringWrapper from '../../common/SubstringWrapper';
+
+const { Menu, MenuItem } = TypeAheadSelect;
+
+const AutoCompleteMenu = ({ results, menuProps }) => {
+  let itemIndex = 0;
+  const grouped = groupBy(results, r => r.category);
+  const items = Object.keys(grouped).sort().map(category => [
```

Comment:
Would it make more sense to return a `React.Fragment` instead `Array`?

Reply:
Yeah, I agree :)